aries-site/src/examples/templates/list-views/ListOrderExample.js

```diff
@@ -26,9 +26,9 @@ export const ListOrderExample = () => {
       role="application"
     >
       <Box gap="small" width="large">
-        <Text weight="bold" size="large">
+        <Heading level={2} size="small" margin="none">
```

Comment:
where is the figma for this change? I clicked on the figma link in the first example but took me to blank list component page

Reply:
Good catch, will connect w a designer.

Reply:
okay thank you